mediacloud/mediawords/util/html.py

```diff
@@ -48,31 +54,32 @@ def __get_meta_refresh_url_from_tag(inner_tag: str, inner_base_url=None) -> Opti
 
         match = re.search(r'content\s*?=\s*?"\d*?\s*?;?\s*?URL\s*?=\s*?\'(.+?)\'', inner_tag, re.I)
         if match:
-            inner_url = match.group(1)
+            inner_url = str(match.group(1))
```

Comment:
is this `bytes` otherwise?

Reply:
fixed by removing the str().

I was trying to avoid a mypy typing complaint here, but I was misunderstanding the root of the problem.